packages/parmap/parmap.1.0-rc10/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+maintainer: "Roberto Di Cosmo <roberto@dicosmo.org>"
+authors: "Roberto Di Cosmo <roberto@dicosmo.org>"
+homepage: "https://github.com/rdicosmo/parmap"
+dev-repo: "https://github.com/rdicosmo/parmap.git"
+bug-reports: "https://github.com/rdicosmo/parmap/issues"
+build: [
+  ["aclocal" "-I" "m4"]
+  ["autoconf"]
+  ["autoheader"]
+  ["./configure"] 
+  [make "DESTDIR=%{prefix}%" "OCAMLLIBDIR=lib" ]
+]
+install: [
+  [make "install" "DESTDIR=%{prefix}%" "OCAMLLIBDIR=lib"]
+]
+remove: [
+  ["aclocal" "-I" "m4"]
+  ["autoconf"]
+  ["autoheader"]
+  ["./configure"] 
+  [make "uninstall" "DESTDIR=%{prefix}%" "OCAMLLIBDIR=lib"]
+]
+depends: [
+  "ocamlfind"
```

Comment:
`Findlib` doesn't seem to be used here. I'd suggest to add a `{build}` attribute. (same for `conf-autoconf` & `conf-aclocal`)

Also is there any ways to avoid using autoconf/aclocal? It looks like the archive linked in the github release not is different from the one you use here and already have autoconf/... ran. See https://github.com/rdicosmo/parmap/releases/tag/1.0-rc10

Reply:
Mmh nope my bad, it's not different in fact but the autoconf & cie. doesn't seem necessary since `./configure` is already present anyway.